### PR TITLE
add Report an Issue link in the README.md

### LINKED ISSUE #353: Modernize plugin: Add a link to “Report an Issue” so that users know the correct location to report bugs, enhancements, and security issues
### What feature do you want to see added?

follow instruction
https://www.jenkins.io/doc/developer/tutorial-improve/add-a-link-to-report-an-issue/

### Upstream changes

_No response_

### PR DESCRIPTION
**Pull request by niranjanib**
Close #353 

### Testing done

This commit has changes only in the README.md, hence no tests were added.

### Proposed changelog entries

Add Report an Issue link in the README.md to report issues in JIRA/Github issue tracker

### Tasks

- [x] Add Report an Issue link in the README.md referring to the correct links in JIRA and Github.

### CODE REVIEW

**Review by MarkEWaite — APPROVED** [review]
Thanks.

@mPokornyETM do you have a preference for one issue tracker over another?  If you'd prefer to have issues come through GitHub, then we could adjust this to only include the link to GitHub.

**Comment by mPokornyETM** [comment]
> 

Hi. 

> Thanks.
> 
> @mPokornyETM do you have a preference for one issue tracker over another? If you'd prefer to have issues come through GitHub, then we could adjust this to only include the link to GitHub.

I personally prefer GitHub. All is in one place and I like when the work is somehow organised.  Therefore I here an project.
Q1: what is prefered way in other plugins?
Q2: Is there someway to migrate all jira issues into Github? 

@niranjanib thx for contributing

**Comment by MarkEWaite** [comment]
> I personally prefer GitHub. All is in one place and I like when the work is somehow organised.

That makes sense to me.  @niranjanib would you like to change the link to only point to GitHub or would you prefer that I do it?

> Q1: what is prefered way in other plugins?

It depends on the plugin.  I prefer Jira for the plugins that I most actively maintain (git plugin, git client plugin, platform labeler plugin) because:
* it is where I started
* it has some features that are valuable that are and not available on GitHub
* it is easier for the security team

> Q2: Is there some way to migrate all jira issues into Github?

@lemeurherve has done a migration of Jira issues to GitHub for the Jenkins infra team and the infra team now uses GitHub issues exclusively.  I don't know the details, but he may be willing to offer insights.

**Comment by mPokornyETM** [comment]
> > I personally prefer GitHub. All is in one place and I like when the work is somehow organised.
> 
> That makes sense to me. @niranjanib would you like to change the link to only point to GitHub or would you prefer that I do it?
> 
> > Q1: what is prefered way in other plugins?
> 
> It depends on the plugin. I prefer Jira for the plugins that I most actively maintain (git plugin, git client plugin, platform labeler plugin) because:
> 
> * it is where I started
> * it has some features that are valuable and not available on GitHub
> * it is a little easier for the security team
> 
> > Q2: Is there some way to migrate all jira issues into Github?
> 
> @lemeurherve has done a migration of Jira issues to GitHub for the Jenkins infra team and the infra team now uses GitHub issues exclusively. I don't know the details, but he may be willing to offer insights.

Thx. In that case I would say.  We use GitHub.  When are here more members and it makes sense switching to jira we can do that anytime. Thx for clarification. 

@niranjanib can you please 🙏 migrate Jira issues also for us. Thx

**Comment by MarkEWaite** [comment]
> @niranjanib can you please 🙏 migrate Jira issues also for us. Thx

I'm confident that @niranjanib does not have permission to migrate issues.  I'm not even sure that @lemeurherve has permissions on this repository to migrate issues.  It would likely mean that @lemeurherve may be able to provide some rough instructions but that you would need to do the migration.

**Comment by mPokornyETM** [comment]
Sorry my mistake.  I used wrong name. My proposal is wdit the PR so that we use GitHub.  Afterwards can be merged and original issue closed. I create new feature requests. And we continue with discussion ther

File: README.md [hunk 1]
```diff
@@ -127,9 +127,13 @@ unclassified:
   for recent versions.
 * See the [old changelog](CHANGELOG.old.md) for versions 2.5 and older.
 
+## Report an Issue
+
+Please report issues and enhancements through the [Jenkins issue tracker](https://www.jenkins.io/participate/report-issue/redirect/#18222) or the [Jenkins issue tracker in GitHub](https://github.com/jenkinsci/lockable-resources-plugin/issues/new/choose)
```
Comment (MarkEWaite) [inline README.md:132]:
```suggestion
Please report issues and enhancements through the [Jenkins issue tracker in GitHub](https://github.com/jenkinsci/lockable-resources-plugin/issues/new/choose)
```

**Comment by mPokornyETM** [comment]
In case, somebody want to continue here : https://issues.jenkins.io/browse/JENKINS-71832?jql=project%20%3D%20JENKINS%20AND%20component%20%3D%20lockable-resources-plugin

**Pull request merged**